{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/urls.py

```diff
@@ -53,6 +53,12 @@
     from django.views import defaults as dj_default_views
     from django.urls import get_callable
 
+    # debug toolbar
+    import debug_toolbar
+    urlpatterns += [
+        url(r'^__debug__/', include(debug_toolbar.urls)),
```

Comment:
I don't think this is possible. `debug_toolbar.urls` is not a module, its just a variable (tuple) in debug-toolbar/__init__.py